### ISSUE TITLE
Incorrect focus highlighting for paragraph starting with a command

### ISSUE THREAD

**Issue by ktraunmueller** [ktraunmueller/Compositor#471]
<img width="776" alt="Screenshot 2019-05-16 at 21 53 06" src="https://user-images.githubusercontent.com/650763/57882876-0b497f80-7825-11e9-97d6-a5123269df85.png">

Source snippet:
```
\begin{itemize}
\item CoreFoundation's \texttt{CFString}
\item Foundation's \texttt{NSString}
\item Swift's native \texttt{String} type as provided by the Swift standard
library (stdlib)
\end{itemize}

\#\#\# 
```

**Comment by ktraunmueller** [comment]
As of 1.22, the source information is correct, but the focus region highlighting highlights the wrong paragraph (the itemize environment in the example, instead of the paragraph with the 3 #'s).

**Comment by ktraunmueller** [comment]
fixed in 1.23
